jwst/datamodels/steppars.py

```diff
@@ -0,0 +1,34 @@
+"""Step parameters model"""
+from copy import copy
+
+from .model_base import DataModel
+
+__all__ = ['StepParsModel']
+
+DEFAULT_META = {
+    'date': 'SPECIFY DATE',
+    'origin': 'STScI',
+    'telescope': 'JWST',
+    'reftype': 'pars-step',
+    'pedigree': 'SPECIFY PEDIGREE',
+    'description': 'Parameters for calibration step SPECIFY',
+    'author': 'SPECIFY AUTHOR',
+    'useafter': 'SPECIFY',
+    'instrument': {
+        'name': 'SPECIFY',
+    },
+}
+
+
+class StepParsModel(DataModel):
+    """
+    A data model for `Step` parameters.
+    """
+    schema_url = "steppars.schema"
+    supported_formats = ['yaml', 'json', 'asdf']
+
+    def __init__(self, init=None, **kwargs):
+        super().__init__(init=init, **kwargs)
+        meta = copy(DEFAULT_META)
+        meta.update(self.meta.instance)
+        self.meta.instance.update(meta)
```

Comment:
Is there a reason this copy-update-update is done rather than relying on the schema to take care of default values?  If a default value is specified in the schema, then it should get populated, right?  Seems like the defaults are located in to places here.

Reply:
And the schema would take care of it. Will remove.

Note: These are leftover from, and still is the case, that the datamodel is not finished.

Reply:
Oh, now I remember. This was done to force the meta to be completely populated. Just having the schema definition was insufficient. Is there a way to get ever required schema element to actually exist in a newly created model?

Reply:
If you define defaults in the schema, then you can access them to get them populated in the instantiated `DataModel`.  It is the reason that there are those
```
    self.data = self.data
    self.dq = self.dq
    self.err = self.err
```
lines in some datamodels that have defaults set for these attributes.